lxd/auth/oidc/oidc.go

```diff
@@ -302,13 +305,18 @@ func getAccessTokenVerifier(issuer string) (op.AccessTokenVerifier, error) {
 
 // NewVerifier returns a Verifier.
 func NewVerifier(issuer string, clientid string, audience string) (*Verifier, error) {
-	cookieKey, err := uuid.New().MarshalBinary()
-	if err != nil {
-		return nil, fmt.Errorf("Failed to create UUID: %w", err)
+	cookieHashKey := securecookie.GenerateRandomKey(32)
```

Comment:
I'd like to see some discussion & comment around why the number 32 was chosen here.

CC @simondeziel

Reply:
Will this change break any existing oidc logics btw @markylaing ?

Reply:
Yes. Users will need to log in again.

Reply:
OK, that really should be highlighted clearly in the PR description as its potentially an issue we need to consider.

Reply:
@tomponline 32 bytes seems to be the required length for AES-256 according to https://github.com/gorilla/securecookie/blob/main/README.md?plain=1#L36C30-L37C60

Reply:
Thanks we should have a comment explaining that for the benefit of future readers to avoid "magic numbers" in code.

Reply:
Oops, I didn't notice this was the hash part. For the hash part, 32 or 64 is supported. I'd go with 64 there since you picked AES-256 anyway.

Reply:
Either way, I think that's a good idea to use `securecookie` mechanism, thanks for doing that!

Reply:
There are now comments to explain the cookie keys, how they are derived, and the required length for security.